src/components/Panels/Guider/Diagram/Diagram.tsx

```diff
@@ -135,19 +152,20 @@ function Flow() {
         id: `${state.m2ComaM1CorrectionsSource}-coma`,
         source: state.m2ComaM1CorrectionsSource ?? '',
         target: 'coma',
-        animated: true,
-        className: 'active',
+        animated: isActive,
+        className: isActive ? 'active' : 'inactive',
       });
     }
 
     if (state.m1CorrectionsEnable) {
       const pos = sourceNodes.findIndex((n) => n.id === state.m2ComaM1CorrectionsSource);
+      const isActive = isSourceActive(state.m2ComaM1CorrectionsSource ?? '');
```

Comment:
Suggestion: have the `issourceActive` function accept `string | undefined` and then you won't need the `?? ''` anymore

Reply:
Will do!